### PR TITLE
added in instructions as well as progress

### PR DESCRIPTION
**Pull request by oliverripps**
@Sammidysam I tried to make how to run this application clear as we did for orb-server as well as adding in a progress file summarizing my progress. I am happy to send the db.php file to anyone who needs it or just send it to Rowan or you or whoever will be working with the person who picks this up.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-docker run -d --name orb-update -p 8000:80 -v $(pwd):/var/www/html orb-status
+docker run -d --name orbs-status -p 8000:80 -v $(pwd):/var/www/html orb-status
```
Comment (Sammidysam) [inline run.sh:3]: Super minor, but maybe we should build the image as named `orbs-status` as well to align with the repo name?

**Review by Sammidysam — APPROVED** [review]
One super super minor comment, but I believe this is all good to go.

**Comment by oliverripps** [comment]
Merging

**Comment by Sammidysam** [comment]
Resolved my comment then since no action was taken.

**Pull request merged**